### PR TITLE
Add a jupyter notebook to AWS TF example

### PR DESCRIPTION
**Pull request by sahil1105**
Basically to give a UI for users.

### CODE REVIEW

**Comment by sahil1105** [comment]
@rkarthik29 FYI

**Comment by sahil1105** [comment]
@alifarhid FYI

File: terraform/aws/README.md [hunk 1]
```diff
@@ -1,52 +1,91 @@
 # Spin up a cluster on AWS for Bodo using Terraform
 
-1. [Install Terraform](https://www.terraform.io/downloads.html).
+1.  [Install Terraform](https://www.terraform.io/downloads.html).
 
-1. Create an AMI. See [this](./bodo-ami/README.md) for instructions. Skip this step if using an existing image.
+1.  Create an AMI. See [this](./bodo-ami/README.md) for instructions. Skip this step if using an existing image.
```
Comment (ritwika314) [inline terraform/aws/README.md:5]:
```suggestion
1.  Create an AMI. See [here](./bodo-ami/README.md) for instructions. **Skip this step if using an existing image.**

File: terraform/aws/README.md [hunk 2]
```diff
@@ -1,52 +1,91 @@
...
 
-1. Set up your AWS credentials. We recommend doing this through [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html), but other methods should also work.
+1.  Set up your AWS credentials. We recommend doing this through [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html), but other methods should also work.
```
Comment (ritwika314) [inline terraform/aws/README.md:7]: Why are all the numbers 1.  ?

Comment (sahil1105) [reply terraform/aws/README.md:7]: It's a markdown thing, it'll render the numbers correctly and you don't need to keep count manually.

File: terraform/aws/README.md [hunk 3]
```diff
@@ -1,52 +1,91 @@
...
 
-    The provided [``bodo_cluster.sh``](./bodo_cluster.sh) script is a wrapper around some common terraform commands we'll use to manage our deployment. Set the above variables in this script.
+    e. `AWS_DEFAULT_REGION` (optional): The region to deploy the cluster in. Default: `us-east-1`.
 
-1. Do a dry-run using:
+    f. `CLUSTER_MEMBERS_COUNT` (optional): Number of instances in the cluster. Default: 2.
+
+    g. `CLUSTER_INSTANCE_TYPE` (optional): Instance type of worker instances. See [this](https://aws.amazon.com/ec2/instance-types/) for a full list. Default: `c5n.xlarge`.
+
+    g. `NOTEBOOK_INSTANCE_TYPE` (optional): Instance type of notebook instance. This instance runs the JupyterLab server, hence it doesn't need to be a large instance. See [this](https://aws.amazon.com/ec2/instance-types/) for a full list. Default: `c5n.xlarge`.
```
Comment (ritwika314) [inline terraform/aws/README.md:25]:
```suggestion
    g. `NOTEBOOK_INSTANCE_TYPE` (optional): Instance type of notebook instance. This instance runs the JupyterLab server, hence it doesn't need to be a large instance. See [here](https://aws.amazon.com/ec2/instance-types/) for a full list. Default: `c5n.xlarge`.

**Review by alifarhid — APPROVED** [review]

**Pull request merged**